test/authorize-specs.js

```diff
@@ -0,0 +1,52 @@
+import { authorize } from '../lib/authorize.js';
+import 'mochawait';
+import * as teen_process from 'teen_process';
+import * as xcode from 'appium-xcode';
+import path from 'path';
+import sinon from 'sinon';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+
+let teenProcessMock = null;
+let xcodeMock = null;
+let pathMock = null;
+let sandbox = null;
+
+describe('Authorize test', () =>{
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();  
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+  
+  it('should throw and error', async () => {
+    teenProcessMock = sandbox.mock(teen_process);
+    teenProcessMock.expects('exec').once().withExactArgs('DevToolsSecurity --enable').throws('foobar');
+    await authorize().should.eventually.be.rejectedWith('foobar');
+    sandbox.verify();
```

Comment:
what @sebv means is, make sure you can get the test to fail by breaking your code. that way you're sure the test is actually testing something. if a test never fails even when the code is broken, it's not doing you any good.